okapi-common/src/main/java/org/folio/okapi/common/logging/FolioLoggingContext.java

```diff
@@ -0,0 +1,70 @@
+package org.folio.okapi.common.logging;
+
+import io.vertx.core.Context;
+import io.vertx.core.Vertx;
+import org.apache.logging.log4j.core.LogEvent;
+import org.apache.logging.log4j.core.config.plugins.Plugin;
+import org.apache.logging.log4j.core.lookup.StrLookup;
+
+/**
+ * This class should be used for storing context variables
+ * and use them in logging events.
+ */
+@Plugin(name = "FolioLoggingContext", category = StrLookup.CATEGORY)
+public class FolioLoggingContext implements StrLookup {
```

Comment:
VertxLoggingContext might be a better name.

Reply:
I think, that there is a lot of stuff related to Vertx and VertxLoggingContext can be jumble with other Vertx classes. Does that make sense?